go.mod

```diff
@@ -43,15 +43,18 @@ require (
 	github.com/Microsoft/go-winio v0.6.2 // indirect
 	github.com/ProtonMail/go-crypto v1.1.6 // indirect
 	github.com/aws/aws-sdk-go-v2/service/internal/accept-encoding v1.12.3 // indirect
+	github.com/aws/aws-sdk-go-v2/service/kms v1.38.3 // indirect
 	github.com/aws/aws-sdk-go-v2/service/secretsmanager v1.35.2 // indirect
 	github.com/aws/aws-sdk-go-v2/service/ssooidc v1.29.1 // indirect
 	github.com/cloudflare/circl v1.6.0 // indirect
+	github.com/cviecco/piv-go/v2 v2.0.0-20250406190009-99a37f723b95 // indirect
```

Comment:
Can we use the upstream repository instead? If needed, try merging upstream and changes we need?

Reply:
Change code, no need for my for and/or upstreaming.